Formula/kumo.rb

```diff
@@ -0,0 +1,24 @@
+class Kumo < Formula
+  desc "Kumo: Word Clouds in Java"
+  homepage "https://github.com/kennycason/kumo"
+  url "https://search.maven.org/remotecontent?filepath=com/kennycason/kumo-cli/1.12/kumo-cli-1.12.jar"
+  sha256 "9d67d943676be57fe75d47f1832c4e8cb415ee8e4e82a71d4d7d1e7b4effe7c4"
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "kumo-cli-1.12.jar"
+    bin.write_jar_script libexec/"kumo-cli-1.12.jar", "kumo"
+    puts "Finished installing kumo 1.12"
+  end
+
+  def server_script(server_jar); <<-EOS.undent
```

Comment:
What is this function trying to achieve? It is never called…

Reply:
Copy-pasta

Reply:
This is left over from a previous example I found within homebrew. I had meant to remove it.